Major Text Change Process

jjhursey opened the pull request:
* Closes #21

jjhursey commented:
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

* You do not need to vote on every proposal
* If you have no opinion, don't vote - that is also useful data
* If you've already commented on this issue, please still vote so
      we know your current thoughts
* Not all proposals solve exactly the same problem, so we may end
      up accepting proposals that appear to have some overlap
This is not a binding majority-rule vote, but it will be a very
significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

* Hooray or Rocket: I support this so strongly that I
      want to be an advocate for it
* Heart: I think this is an ideal solution
* Thumbs up: I'd be happy with this solution
* Confused: I'd rather we not do this, but I can tolerate it
* Thumbs down: I'd be actively unhappy, and may even consider
      other technologies instead
If you want to explain in more detail, feel free to add another
comment, but please also vote on this comment.

abouteiller commented on pmix_governance.tex:
```diff
@@ -1092,5 +1092,124 @@ \subsubsection{Deprecation}\label{deprecation}}
+
+The process for major text changes begins with a GitHub Draft PR while writing the proposal, optionally proceeded by a GitHub Issue for more general discussion and guidance.
+When the GitHub PR is ready for discussion the authors(s) must remove the Draft PR status, add the ``RFC'' and ``Major Text Change'' labels, and add the Straw Poll Comment to it.
+Then the author(s) must send an announcement of the PR to the PMIx community mailing list requesting consideration of the PR as an errata change.
```
as a major text  change

jjhursey replied:
Fixed. Thanks!

kathrynmohror commented:
Line 610: I suggest changing "All proposed additions to the PMIx Standard enter the PMIx Standard
  under Provisional status. " to something like "All proposed API or attribute additions to the PMIx Standard enter the PMIx Standard under Provisional status. "

Line 719: I suggest changing "Discussion of the PR will continue until all dissenting opinions and
questions have been addressed, upon which the PR may be put forward for
Provisional acceptance. " to something like " Discussion of the PR will continue until all dissenting opinions and
questions have been addressed, upon which the PR may be put forward for formal decision by the ASC."

mike0042 commented on pmix_governance.tex:
```diff
@@ -1092,5 +1092,124 @@ \subsubsection{Deprecation}\label{deprecation}}
+changes to descriptions of APIs including advice to implementors/users.
+
+The process for major text changes begins with a GitHub Draft PR while writing the proposal, optionally proceeded by a GitHub Issue for more general discussion and guidance.
+When the GitHub PR is ready for discussion the authors(s) must remove the Draft PR status, add the ``RFC'' and ``Major Text Change'' labels, and add the Straw Poll Comment to it.
```
author(s)

jjhursey replied:
Fixed. Thanks!

jjhursey commented:
@mike0042 and @kathrynmohror I addressed your comments in the last two commits. Note that I also rebased this PR onto master to pick up the 'draft' watermark.

The current PDF rendering is here: 
[pmix_governance.pdf](https://github.com/pmix/governance/files/6320469/pmix_governance.pdf)

jjhursey commented:
PMIx ASC 2Q 2021: May 11 & May 13
 * Passed first vote: 7 yes / 0 no / 1 abstain
 * Eligible for second vote 3Q 2021 ASC Meeting

The new commit is a typo fix found during the reading.

jjhursey commented:
PMIx ASC 3Q 2021
 * Passed the second vote: 8 yes / 0 no / 1 abstain
 * Will be merged into the next release

jjhursey commented:
I rebased and merged this PR. It'll be in the v1.5 version of the document.
